framework/src/mfem/functormaterials/MFEMGenericFunctorVectorMaterial.C

```diff
@@ -5,6 +5,59 @@
 
 registerMooseObject("MooseApp", MFEMGenericFunctorVectorMaterial);
 
+/// Handle any numerical vector values, which should be enclosed in curly braces
+std::vector<MFEMVectorCoefficientName>
+processLiteralVectors(const std::vector<MFEMVectorCoefficientName> & input)
+{
+  std::vector<MFEMVectorCoefficientName> result;
+  bool in_literal = false;
+  MFEMVectorCoefficientName literal;
+  for (const auto & item : input)
+  {
+    if (in_literal)
+    {
+      if (item.front() == '{')
+      {
+        mooseError("Nested numeric vector values are not permitted in "
+                   "MFEMGenericFunctorVectoMaterial prop_values.");
+      }
```

Comment:
```suggestion
        mooseError("Nested numeric vector values are not permitted in "
                   "MFEMGenericFunctorVectoMaterial prop_values.");
```

Reply:
Actually, seeing as you've now committed these in #30803 we might as well just merge that.